index.html

```diff
@@ -140,8 +140,27 @@ <h3>LET'S CONNECT</h3>
       <li class="language-item"><img src="./Bullets/html.png" alt="html-logo"><p>HTML</p></li>
       <li class="language-item"><img src="./Bullets/css.png" alt="css-logo"><p>CSS</p></li>
       <li>Frameworks<hr></li>
-      <li>Skills</li>
+      <li>Skills<hr></li>
     </ul>
-  </section>
+  </section><br>
+  <form class="form" action="https://formspree.io/f/xdobanld" method="POST">
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <ul>
+      <li>
+        <label for="name"></label>
+        <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+      </li>
+      <li>
+        <label for="email"></label>
+        <input type="email" name="user_mail" id="email" placeholder="youremail@email.com" required>
+      </li>
```

Comment:
Please try to follow the details on the figma's design 📝 Your inputs for name and email have a red border. What do you think if we change those borders to meet the project requirements 🙌😎

#### Your form 
![Captura](https://user-images.githubusercontent.com/15807343/149032543-6ae0e4fe-dc06-476a-8e3e-dce2d0d91051.PNG)

#### Figma's 
![image](https://user-images.githubusercontent.com/15807343/149032576-ad434cef-377b-4fa4-ad88-41038c5ea365.png)